week-1/L-functions-nested/exercise.js~

```diff
@@ -0,0 +1,17 @@
+/*
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+*/
+
+
+function mentorsStudents(ments, studs) {
+  var percentagemen = (ments / studs) * 100;
+  var percentstud = (studs / ments ) * 100;
+  return percentstud
```

Comment:
This is only returning percentage of students 
try declaring another function which logs the percentage when called
Call the function within mentorsStudents function

Reply:
> This is only returning percentage of students
> try declaring another function which logs the percentage when called
> Call the function within mentorsStudents function

Hello Diya, this is showing old code for some reason. The function has gone through a few changes and now returns an array like this:`return [permen, perstu]`